tests/hla/process_randomlist.py

```diff
@@ -0,0 +1,204 @@
+                    update_hdr_wcs=False,print_fit_parameters=True,print_git_info=False,output=False)
+
+                # Filtered datasets
+                if datasetTable['doProcess'].sum() == 0:
+                    print("TEST_RANDOM. Filtered Dataset: ", dataset, "\n")
+                    numProcessedDatasets -= 1;
+                # Datasets to process
+                elif datasetTable['doProcess'].sum() > 0:
+                    # Determine images in dataset to be processed and the number of images
+                    # This is in case an image was filtered out (e.g., expotime = 0)
+                    index = np.where(datasetTable['doProcess']==1)[0]
+                    fitQual = datasetTable['fit_qual'][index[0]]
+
+                    # Update the table with the datasetKey which is really just a counter
+                    datasetTable['datasetKey'][:] = datasetKey
+                    datasetTable['completed'][:] = True
+                    datasetTable.write(outputName, format='ascii.ecsv')
+                    #datasetTable.pprint(max_width=-1)
+   
+                    # Successful datasets
+                    if (fitQual <= 2):
+                        print("TEST_RANDOM. Successful Dataset (fit_qual <= 2): ", dataset, "\n")
+                        numSuccess += 1
+                    elif 2 < fitQual <= 4:
+                        print("TEST_RANDOM. Qualified Successful Dataset (2 < fit_qual <= 4): ", dataset, "\n")
+                        numQualSuccess += 1
+                    # Unsuccessful datasets
+                    else:
+                        print("TEST_RANDOM. Unsuccessful Dataset (fit_qual = 5): ", dataset, "\n")
+                        numUnsuccessful += 1
+
+                # Append the latest dataset table to the summary table 
+                allDatasetTable = vstack([allDatasetTable, datasetTable])
+
+            # Catch anything that happens as this dataset will be considered a failure, but
+            # the processing of datasets should continue.  Generate sufficient output exception
+            # information so problems can be addressed.
+            except Exception:
+           
+                exc_type, exc_value, exc_tb = sys.exc_info()
+                traceback.print_exception(exc_type, exc_value, exc_tb, file=sys.stdout)
+                print("TEST_RANDOM. Exception Dataset: ", dataset, "\n")
+                numException += 1
+                continue
+
+        # Perform some clean up
+        if os.path.isfile('ref_cat.ecsv'): 
+            os.remove('ref_cat.ecsv')
+        if os.path.isfile('refcatalog.cat'):  
+            os.remove('refcatalog.cat')
+        for filename in os.listdir():
+            if filename.endswith('flt.fits') or filename.endswith('flc.fits'):
+                os.unlink(filename)
+
+        # Write out the table
+        allDatasetTable.write('resultsBigTest.ecsv', format='ascii.ecsv')
```

Comment:
Would it be more useful for comparing with other runs to have a date-time-stamped filename instead of a fixed generic filename?  Admittedly, a time-unique name would definitely not be needed just for running the test under pytest, so this is merely a suggested issue to consider when time permits.

Reply:
I should have done this from the start.  Updated using seconds since the epoch.